src/plugins/intel_npu/src/plugin/npuw/util.hpp

```diff
@@ -141,6 +141,14 @@ Impl<M> _(std::shared_ptr<M> pM) {
 
 }  // namespace at
 
+// Written here to be a drop-in replacement for ov::parallel_for for the debug purposes
+template <typename F>
+void non_parallel_for(std::size_t count, F&& f) {
```

Comment:
Are we keeping it?

Reply:
Yes, moved here to stop reinventing it all the time